src/settings-ui/Settings.UI.Library/ViewModels/PowerLauncherPluginViewModel.cs

```diff
@@ -110,6 +115,9 @@ public int WeightBoost
                     NotifyPropertyChanged();
                     NotifyPropertyChanged(nameof(ShowNotAccessibleWarning));
                     NotifyPropertyChanged(nameof(ShowNotAllowedKeywordWarning));
+                    NotifyPropertyChanged(nameof(ShowMathematicKeywordWarning));
```

Comment:
Does changing WeightBoost affect these anyhow?

Reply:
Why should it? The warning is only shown if the condition is true.

I only added the notify where others for the activation keyword exists already.

Reply:
Let's remove it then?

Reply:
Remove the other two (`accessible` and `NotAllowedKeyword`) too?